src/consortiaServices.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
+
+export function isUserInCentralTenant(stripes) {

```

Comment:
I agree with this name too, but can we rename it to `checkIfUserInCentralTenant` to make it easier to name after import?

`const isUserInCentralTenant = checkIfUserInCentralTenant(stripes);`

Reply:
🤷 `const isCentralTenantUser = isUserInCentralTenant(stripes);`

Reply:
@zburke it's going to give a lint error:
![image](https://github.com/folio-org/stripes-core/assets/19309423/0d7edd4a-b378-49af-b990-82b7c281b095)

Reply:
I don't understand. I suggested slightly different names for the variable and the function specifically to avoid the scope overlap. But I don't have strong feelings here, and I'm sorry if weighing in has muddied the water. 

I generally like the convention of using verbs at the start of functions that return boolean values, e.g. `hasPermission` or `isAuthorized` or `isInCentralTenant`, but that's just a nice convention, not a rule that must be followed. You have to remember, I learned to program with vi during the previous millennium, before intellisense or code-completion 👴 . Back then it was super helpful to have these kinds of conventions because they saved you from having to look up the source to find out, "Does this return false on failure or throw an exception?" `isValid(stuff)` was a good hint that it would return a boolean.